README.md

```diff
@@ -54,6 +54,13 @@ We use the `main` branch to develop new features. For production code, see the
 
 <!-- markdown-link-check-enable -->
 
+### Claims/Policies development
+
+You need a policy that is linked to your account in order to create a claim. You can create a policy for your account by
+running:
+
+`yarn create-user-policy --address <address> --coverage-amount 500 --ipfs-hash some-hash-001`
```

Comment:
This doesn't work. It needs to be added to package.json (I see this is added in the other PR though).

Reply:
I got a bit ahead of myself here, but ja, the other PR needs to be merged into main, and then I will also get rid of the last uses of REACT_APP_CLAIMS_MANAGER_ADDRESS (which was introduced with that branch) 👍